### PR TITLE
completed ncnews-8

### PR DESCRIPTION
**Pull request by Issy786**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -38,3 +38,11 @@ export const patchArticle = (article_id) => {
     { inc_votes: 1 }
   );
 };
+
+export const fetchAllCommentsByArticleId = (article_id) => {
+  return fetch(
```
Comment (xmenbteam) [inline src/api.js:43]: Get on Axios if you can please!

File: src/components/comments.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      setComments(data.comments);
+    });
+  }, [article_id]);
+
+  return (
+    <div className="comments">
+      <h3 className="comments-header">Comments</h3>
+
+      {comments.map((comment) => {
+        return (
```
Comment (xmenbteam) [inline src/components/comments.jsx:18]: Make this a separate <Comment /> component for ultimate reusability please :)

**Review by xmenbteam — APPROVED** [review]

**Comment by xmenbteam** [comment]
Nice! Well done. Please think about using Axios rather than fetch. It's not a lot of work and it'll make your life easier :)

File: src/components/comments.jsx (hunk 2 above)
Comment (Issy786) [reply src/components/comments.jsx:18]: Thanks for the feedback.. I will make those changes :)

**Pull request merged**